src/App.jsx

```diff
@@ -31,32 +42,61 @@ const App = () => {
 
   const productsList = products.map((product) => {
     return (
-      <div key={product.id}>
-        {product.name} - {`${product.price} kr `}
-        <button
-          data-cy="add-to-order-button"
-          onClick={() => addToOrder(product.id)}
-        >
-          Add to Order
-        </button>
-      </div>
+      <>
+        <ProductsListItem
+          product={product}
+          addToOrder={addToOrder}
+        />
+        <Divider variant="inset" component="li" />
+      </>
     );
   });
 
   return (
     <>
-      <h1>Slowfood</h1>
-      {order && (
-        <button data-cy="view-order" onClick={() => setViewOrder(!viewOrder)}>
-          {viewOrder ? "Hide Order" : "View Order"}
-        </button>
-      )}
-      <h3 data-cy="message-box">{message}</h3>
-      {viewOrder ? (
-        <OrderDetails order={order} />
-      ) : (
-        <div data-cy="product-list">{productsList}</div>
-      )}
+      <Header
+        brand="Material Kit React"
+        fixed
+        color="transparent"
+        changeColorOnScroll={{
+          height: 400,
+          color: "white",
+        }}
+      />
+      <Parallax image={require("./assets/img/bg4.jpg").default}>
+        <div className={classes.container}>
+          <GridContainer>
+            <GridItem>
+              <div className={classes.brand}>
+                <h1 className={classes.title}>Material Kit React.</h1>
```

Comment:
?? Material Kit React?

Reply:
It's the name of the Material UI theme. It needs to be changed of course